converter/src/main/java/org/dswarm/converter/flow/CSVSourceResourceTriplesFlow.java

```diff
@@ -46,14 +46,11 @@ private CSVSourceResourceTriplesFlow(
 	}
 
 	@AssistedInject
-	private CSVSourceResourceTriplesFlow(
-			@Assisted final Configuration configuration) throws DMPConverterException {
+	private CSVSourceResourceTriplesFlow(@Assisted final Configuration configuration) throws DMPConverterException {
 		super(configuration);
 	}
 
-	@AssistedInject
-	private CSVSourceResourceTriplesFlow(
-			@Assisted final DataModel dataModel) throws DMPConverterException {
+	protected CSVSourceResourceTriplesFlow(final DataModel dataModel) throws DMPConverterException {
```

Comment:
Ok, this might be a bit hacky. I only wanted the monitoring when the data was persisted, not when the scheme is created (as we discussed yesterday). So, there two triple flows of which only one monitors and the rule of thumb is, when a flow is created from a data model, it shall be monitored. This constructor exists only for its subclass. I hope this is somewhat understandable or maybe I should provide some documentation hints (doc issue is still open though 😉)


> On Mar 27, 2015, at 10:03 AM, Bo Ferri <notifications@github.com> wrote:
> 
> In converter/src/main/java/org/dswarm/converter/flow/CSVSourceResourceTriplesFlow.java:
> 
> >  		super(configuration);
> >  	}
> >  
> > -	@AssistedInject
> > -	private CSVSourceResourceTriplesFlow(
> > -			@Assisted final DataModel dataModel) throws DMPConverterException {
> > +	protected CSVSourceResourceTriplesFlow(final DataModel dataModel) throws DMPConverterException {
> y is this the other way around then above?
> 
> —
> Reply to this email directly or view it on GitHub.
>